internal/internal_worker_test.go

```diff
@@ -445,7 +445,7 @@ func (s *internalWorkerTestSuite) testCompleteActivityHelper(opt *ClientOptions)
 }
 
 func (s *internalWorkerTestSuite) TestCompleteActivity() {
-	s.testCompleteActivityHelper(nil)
+	s.testCompleteActivityHelper(&ClientOptions{IsolationGroup: ""})
```

Comment:
should probably have a value or not be a field at all, since this is indistinguishable from `&ClientOptions{}`

Reply:
I think I just added it beccause some tests were using methods on the options and panicing